awx_collection/tests/integration/targets/tower_inventory/tasks/main.yml

```diff
@@ -25,17 +26,30 @@
         that:
           - "result is changed"
 
+    - name: Create an Instance Group
+      tower_instance_group:
+        name: "{{ group_name1 }}"
+        state: present
+      register: result
```

Comment:
This needs to be something like `instance_result`, otherwise the `insights_credential: "{{ result.id }}"` line in the following task can't find the proper Insights credential